src/helics/application_api/ValueFederateManager.cpp

```diff
@@ -437,6 +437,40 @@ bool ValueFederateManager::getPublicationOption (const Publication &pub, int32_t
     return coreObject->getHandleOption (pub.handle, option);
 }
 
+std::string ValueFederateManager::getInputInfo(const Input &imp) const {
```

Comment:
We should be able to deal with this directly in the Value Federate since the info is contained in the core object no reason for the extra layer of calls here.